x-pack/plugins/alerting/server/task_runner/types.ts

```diff
@@ -9,6 +9,7 @@ import { KibanaRequest, Logger } from '@kbn/core/server';
 import { ConcreteTaskInstance } from '@kbn/task-manager-plugin/server';
 import { PublicMethodsOf } from '@kbn/utility-types';
 import { ActionsClient } from '@kbn/actions-plugin/server/actions_client';
+import { DecoratedError } from '@kbn/task-manager-plugin/server/task_running';
```

Comment:
nit: should we export this from the `task_manager/server/index.ts` file so we can just import from `@kbn/task-manager-plugin/server`?